modules/macros/pages/link-macro-attribute-parsing.adoc

```diff
@@ -1,47 +1,45 @@
 = Link Macro Attribute Parsing
 
 If named attributes are detected between the square brackets of a link macro (including URL macros), that text is parsed as an attribute list.
-This page explains the conditions when this occurs and how to write the linked text so it does not get caught up in this parsing.
+This page explains the conditions when this occurs and how to write the link text so it does not get caught up in this parsing.
 
 == Linked text alongside named attributes
 
-Normally, the whole text between the square brackets of a link macro is treated as the linked text.
+Normally, the whole text between the square brackets of a link macro is treated as the link text.
 
 [source]
 ----
 https://discuss.asciidoctor.org[Discuss Asciidoctor]
 ----
 
-However, if the text contains both an equals sign (`=`) and a comma (`,`), and the comma comes after the equals sign, the text is parsed as an attribute list.
-In this case, the linked text is the first positional attribute.
+However, if the text contains an unescaped equals sign (`=`), the text is parsed as a comma-separated attribute list.
+If the first attribute, i.e. the text before the first comma, has no equals sign, the link text is this first, positional, attribute.
 
-The following example shows a URL macro with custom linked text alongside named attributes.
+The following example shows a URL macro with custom link text alongside named attributes.
 
 [source]
 ----
 https://discuss.asciidoctor.org[Discuss Asciidoctor,role=resource,window=_blank]
 ----
 
-If you want to define a single named attribute, but leave the linked text empty (so it falls back to the URL), then you need to start the attribute list with a comma.
```

Comment:
It seems like we lost this example. I think it's important to explain what happens when the link text is empty and other attributes are present as this is a valid use case.